config/values/30-istio-values.yml

```diff
@@ -0,0 +1,11 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+istio:
+  #! build_version is updated by build/istio/build.sh, do not edit it manually
+  build_version: "1.6.4"
```

Comment:
There is a concern that this exposes the apparent istio version to the operator - they may think they can change it via this value. Perhaps it could be insulated from the user in your library eval value replacement step or replaced in another way?

Reply:
Is there a place where we can put "values used by multiple overlays but not editable by users?" We need this in a few places (changing it and rerunning our scripts is actually how we upgrade istio), but we agree we don't want users to think they can edit it.

The same goes for the image shas below, and the istiod replicas. These are internal values that users shouldn't be editing but need to be in one place to DRY up our overlays.

Reply:
We've now eliminated the need for these values:

* `build_version` -- we defined a function in the file it's used to return the value, and updated our build script to update its value
* `istiod.replicas` -- hardcoded while we wait for https://github.com/cloudfoundry/cf-for-k8s/pull/427
* `upgrade_sidecars.image` -- hardcoded with a future chore for our image pipeline to update it in place (this one doesn't change much)